lesson04/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Lesson04 - Assignment 1 - Extra</title>
+</head>
+
+<body>
+  <h1>This is Lesson4 - Assignment 1 - Regular</h1>
+  <p>Write a simple webpage opens an alert that takes a number from the user and squares it and opens an alert with the result. Create an HTML page and write your javascript in a script tag, or create a JavaScript file and link to it from the HTML
+    page.</p>
+  <p>You must use a function and an event handler.</p>
+
+  <button onclick="myFunction()">What's the result?</button>
+  <p id="test"></p>
+  <script>
```

Comment:
Nice! I like the added bonus =) . But you can move all this code into your JS file